src/NServiceBus.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -174,74 +174,53 @@ void Connection_ConnectionShutdown(object sender, ShutdownEventArgs e)
 
                 if (rejectMessage)
                 {
-                    await RejectMessage(channel, message.DeliveryTag, messageId).ConfigureAwait(false);
+                    await RejectMessage(channel, message.DeliveryTag).ConfigureAwait(false);
                 }
                 else
                 {
-                    await AcknowledgeMessage(channel, message.DeliveryTag, messageId).ConfigureAwait(false);
+                    await AcknowledgeMessage(channel, message.DeliveryTag).ConfigureAwait(false);
                 }
             }
+            catch (AlreadyClosedException ex)
+            {
+                Logger.Warn($"Attempt to acknowledge/reject message {messageId} failed because the channel was closed. The message will be requeued.", ex);
```

Comment:
Does it matter that the log message has changed from explicitly say "acknowledge" or "reject" depending on which operation was attempted to "acknowledge/reject"?

If we care about this, it would be fairly simple to lift a version of this catch into each branch of the `if`.